src/commands/analytics/display-analytics.ts

```diff
@@ -6,45 +6,32 @@ import contrib from 'blessed-contrib'
 
 import { logger } from '@socketsecurity/registry/lib/logger'
 
+import { fetchOrgAnalyticsData } from './fetch-org-analytics.ts'
+import { fetchRepoAnalyticsData } from './fetch-repo-analytics.ts' // Note: Widgets does not seem to actually work as code :'(
```

Comment:
☝️ Widgets comment needs to be moved down

Reply:
yeah that one is annoying. the autofix ignores it.